packages/shared/components/ProposalQuestion.svelte

```diff
@@ -2,46 +2,48 @@
     import { Text, FontWeight, Icon, ProposalAnswer } from 'shared/components'
     import { Icon as IconEnum } from '@auxiliary/icon'
     import type { AnswerStatus, Question } from '@iota/wallet'
+    import { createEventDispatcher } from 'svelte'
 
-    export let currentVote: AnswerStatus[] = undefined
+    const dispatch = createEventDispatcher()
+
+    export let allVotes: AnswerStatus[] = undefined
     export let questionIndex: number = undefined
     export let isOpened = false
     export let question: Question
-    export let selectedAnswerValues: number[] // TODO, maybe should be a svelte store
-    export let onClick: () => unknown = () => {}
+    export let selectedAnswerValue: number = undefined
+    export let votedAnswerValue: number = undefined
 
     let percentages: string[]
-    let voteValue: number // TODO: get the current answer being voted by the account
 
     $: answers = [...question?.answers, { value: 0, text: 'Abstain', additionalInfo: '' }]
-    $: showMargin = isOpened || ((voteValue || voteValue === 0) && !isOpened) // voteValue 0 corresponds to abstained vote
-    $: currentVote, setPercentagesFromAccumulated()
+    $: showMargin = isOpened || ((votedAnswerValue || votedAnswerValue === 0) && !isOpened) // votedAnswerValue 0 corresponds to abstained vote
```

Comment:
Might be worth making a constant e.g. `ABSTAIN_ANSWER_VALUE` so we can remove this comment and use it elsewhere as needed.